README.md

```diff
@@ -21,9 +20,13 @@ Install ODO extension manually
 5 Restart the node server inside PFE container to load the ODO extension
 Note: We already had installer for ODO extension, the installer will help you automatically install ODO extension if you are running Codewind on OKD/OpenShift cluster, this is only workaround when installer doesn't work properly
 
-
-Current Limitations
+## Current Limitations
 - ODO extension is supported on Codewind for Eclipse Che with OKD/OpenShift cluster
```

Comment:
ODO extension is supported on Codewind for Eclipse Che with an OKD/OpenShift cluster